frontend/src/pages/admin/access/ManageAccessRequests.tsx

```diff
@@ -1,114 +1,129 @@
+import * as React from 'react';
+import { Container, Row, Col, Form } from 'react-bootstrap';
+import { AccessRequestStatus } from 'constants/accessStatus';
+import { Table } from 'components/Table';
+import { IPaginate, toAccessRequestApiPaginateParams } from 'utils/CommonFunctions';
+import * as API from 'constants/API';
+import { IAccessRequest } from 'interfaces';
+import './ManageAccessRequests.scss';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
The more I see us use redux the more I'm questioning why we're using where we use it.  Does this page share state with other pages?  Or is simply how we're sharing state with sub-components of this larger component.  If it's the latter I suspect we're not entirely designing this correctly.  However I'm still not up-to-speed with everything react/redux and don't have a strong opinion yet.